packages/main/src/WheelSlider.js

```diff
@@ -163,10 +172,40 @@ class WheelSlider extends UI5Element {
 		}
 	}
 
+	_itemIndex(id) {
+		return Number(id.replace(/^\D+/g, ""));
+	}
+
+	_timesMultipliedOnCyclic() {
+		const minElementsInCyclicWheelSlider = 70;
+		const repetition = Math.round(minElementsInCyclicWheelSlider / this._items.length);
+		const minRepetitionCount = 3;
+
+		if (repetition < minRepetitionCount) {
+			return minRepetitionCount;
+		}
+
+		return repetition;
+	}
+
 	get items() {
 		return this._items || [];
 	}
 
+	get showItems() {
```

Comment:
Getters are supposed to be pure functions. It's an anti-pattern to change state in a getter. You should either have a getter, or _showItems, but not both. Please think of an elegant way to remove one or the other.

Reply:
Done